python/pyarrow/_parquet.pyx

```diff
@@ -1278,11 +1297,43 @@ cdef shared_ptr[WriterProperties] _create_writer_properties(
 
     if isinstance(use_byte_stream_split, bool):
         if use_byte_stream_split:
-            props.encoding(ParquetEncoding_BYTE_STREAM_SPLIT)
+            if column_encoding is not None:
+                raise ValueError(
+                    "'column_encoding' is not None")
+            else:
+                props.encoding(ParquetEncoding_BYTE_STREAM_SPLIT)
     elif use_byte_stream_split is not None:
         for column in use_byte_stream_split:
-            props.encoding(tobytes(column),
-                           ParquetEncoding_BYTE_STREAM_SPLIT)
+            if column_encoding is None:
+                column_encoding = {column: 'BYTE_STREAM_SPLIT'}
+            elif column_encoding.get(column, None) is None:
+                column_encoding[column]='BYTE_STREAM_SPLIT'
```

Comment:
Hmm, can you ensure the code is PEP8-compliant?
You should be able to do so with `archery lint --python`.

(here, spaces should be added around `=`)

Reply:
I should have run archery, will do it again. Thanks for pointing it out!